x/crosschain/simulation/operations.go

```diff
@@ -0,0 +1,462 @@
+package simulation
```

Comment:
We should be able to add unit tests for the functions in this file. 
I will be adding that in a separate pr , so as to not increase the number of files for review .

Unit tests would need us to setup an  `App.` Something similar to the `NewSimApp` function.